backend/lib/utils.ts

```diff
@@ -1,9 +1,13 @@
-/* eslint-disable no-console */
-import crypto from "crypto"
```

Comment:
De ce que j'ai trouvé `Math.random()` est pas super safe pour la generation de tokens (1). 
Pour garder la generation avec `"crypto"` tu peux faire un truc comme ça plutôt : 

```
    const randomIndex = crypto.randomInt(ALPHABET.length);
    tokenArray.push(ALPHABET.charAt(randomIndex));
```

(1): https://stackoverflow.com/questions/5651789/is-math-random-cryptographically-secure

Reply:
J'avoue que la sécurité cryptographique de la méthode utilisée pour la génération d'entier aléatoire ne m'avait pas effleuré l'esprit dans ce contexte mais ok pour utiliser `crypto`, d'autant que la lib est déjà présente historiquement